### PR TITLE
hsd-cli: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/hsd-cli.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`hsd-cli info`
+
+- Broadcast a local transaction:
+
+`hsd-cli broadcast {{transaction_hex}}`
+
+- Retrieve a mempool snapshot:
+
+`hsd-cli mempool`
```
Comment (sbrl) [inline pages/common/hsd-cli.md]: Is there any way of specifying which snapshot you want, or is ti automatic?

Comment (pxgamer) [reply pages/common/hsd-cli.md]: You basically get a JSON array of all transactions in the mempool at the current time.

Comment (sbrl) [reply pages/common/hsd-cli.md]: Ahhh ok :-)

**Comment by pxgamer** [comment]
Thanks @mebeim 👍

**Pull request merged**